uiowa/uiowa
# lazy load working with photoswipe

Had to create a MR with two existing patches and modify it to work with D9. COOL BEANS. Created a patch off of merge request so nothing will change. 
Closes #4783
closes #4792 



# How to test


1. composer install
2. blt ds --site=brand.uiowa.edu
3. go to https://brand.uiowa.ddev.site/showcase and make sure images are as lazy (lazyloaded) as Izzy 🐶
